pkgs/by-name/sd/SDL2_image/package.nix

```diff
@@ -36,16 +37,20 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
   ];
 
-  buildInputs = [
-    SDL2
-    giflib
-    libXpm
-    libjpeg
-    libpng
-    libtiff
-    libwebp
-    zlib
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ Foundation ];
+  buildInputs =
+    [
+      SDL2
+      giflib
+      libXpm
+      libtiff
+      libwebp
+      zlib
+    ]
+    ++ lib.optionals (!enableSTB) [
+      libjpeg
+      libpng
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ Foundation ];
```

Comment:
`Foundation` can be dropped, see also https://github.com/NixOS/nixpkgs/pull/398707